packages/beacon-node/src/eth1/eth1DepositDataTracker.ts

```diff
@@ -1,6 +1,6 @@
 import {phase0, ssz} from "@lodestar/types";
 import {ChainForkConfig} from "@lodestar/config";
-import {BeaconStateAllForks, becomesNewEth1Data} from "@lodestar/state-transition";
+import {BeaconStateAllForks, CachedBeaconStateAllForks, becomesNewEth1Data} from "@lodestar/state-transition";
```

Comment:
at some points (once finalized state has `state.eth1_deposit_index >= state.deposit_receipts_start_index`?), this class should stop polling eth1, i.e skip `this.update()` in `runAutoUpdate()`

Reply:
> at some points (once finalized state has `state.eth1_deposit_index >= state.deposit_receipts_start_index`?), this class should stop polling eth1, i.e skip `this.update()` in `runAutoUpdate()`

Yea I am thinking of listening to forkChoiceFinalized, and stop `runAutoUpdate()` if there is a 6110 state that matches the condition. 
Debating whether to do the listening in `BeaconChain` and pass down a stop signal to `Eth1DepositDataTracker`, or just listen directly in `Eth1DepositDataTracker`